pkgs/servers/home-assistant/custom-components/garmin_connect/default.nix

```
@@ -0,0 +1,31 @@
+{
+  lib,
+  buildHomeAssistantComponent,
+  fetchFromGitHub,
+  python3Packages,
```

You can't mix package sets here.

```suggestion
  garminconnect,
  tzlocal,
```